ports/nrf/supervisor/internal_flash.c

```diff
@@ -68,81 +67,14 @@ uint32_t supervisor_flash_get_block_size(void) {
 }
 
 uint32_t supervisor_flash_get_block_count(void) {
-    return ((uint32_t) __fatfs_flash_length) / FILESYSTEM_BLOCK_SIZE ;
-}
-
-#ifdef BLUETOOTH_SD
-STATIC void sd_flash_operation_start(void) {
-    sd_flash_operation_status = SD_FLASH_OPERATION_IN_PROGRESS;
-}
-
-STATIC sd_flash_operation_status_t sd_flash_operation_wait_until_done(void) {
-    while (sd_flash_operation_status == SD_FLASH_OPERATION_IN_PROGRESS) {
-        sd_app_evt_wait();
-    }
-    return sd_flash_operation_status;
-}
-#endif
-
-void supervisor_flash_flush(void) {
```

Comment:
`supervisor_flash_flush()` is called from `supervisor/shared/filesystem.c`, but I don't see it in your refactored code.

Reply:
Yep, see above :-(